lib/features/login/data/network/oidc_error.dart

```diff
@@ -2,4 +2,6 @@ class CanNotFoundOIDCAuthority implements Exception {}
 
 class CanNotFoundOIDCLinks implements Exception {}
 
-class CanNotFoundToken implements Exception {}
\ No newline at end of file
+class CanNotFoundToken implements Exception {}
```

Comment:
```suggestion
class CanNotFindToken implements Exception {}
```
To be gramatically correct.

Reply:
Done